src/common/BaseAnnotator.ts

```diff
@@ -111,6 +113,11 @@ export default class BaseAnnotator {
         this.rootEl.classList.add('ba');
     }
 
+    deleteAnnotation = (annotationId: string): void => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        this.store.dispatch<any>(store.deleteAnnotationAction(annotationId));
```

Comment:
I think `dispatch(store.deleteAnnotationAction(annotationId))` should work fine here without `<any>` since it's not an async action.